Lib/pywrapper.py

```diff
@@ -154,7 +154,7 @@ def grid(axis_ids, latitude=None, longitude=None,
         nvert = 0
         if not type in ['d', 'f', 'i', 'l']:
```

Comment:
the word "type" is a built in python method and should not be used for variable names. Although its technically not a reserved word, this is a bad style choice. I know the people that originally wrote this code arent here any more, but that doesnt mean it cant be fixed!

Reply:
@sterlingbaldwin @durack1 
I have changed the 'type' variable in pywrapper.py to 'data_type' to not shadow the built-in 'type' in Python.  This also involved changing the parameters of some functions to reflect this change.  I will add these changes to the CMOR documentation site.

Reply:
Thanks, I know it may seem small, but this kind of thing can cause serious headaches.